app/src/main/java/com/eveningoutpost/dexdrip/utils/LibreTrendPoint.java

```diff
@@ -0,0 +1,49 @@
+package com.eveningoutpost.dexdrip.utils;
+
+import com.eveningoutpost.dexdrip.Models.GlucoseData;
+import com.eveningoutpost.dexdrip.Models.UserError.Log;
+
+//This class represents a per minute data from the libre.
+public class LibreTrendPoint {
+    long sensorTime; // The number of minutes from sensor start.
+    public long rawSensorValue; // The raw value of the sensor
+    int flags; // The flags that was received from the sensor (if any).
+    public GlucoseData.DataSource source; // Did we read this data from
+                                          // bluetooth or NFC? This affects the
+                                          // way that flags are interputed.
+
+    static final String TAG = "LibreTrendPoint";
+
+
+    public LibreTrendPoint(){}
+
+    // For testing
```

Comment:
What is meant with "For testing"? If you like to test something, please provide Unit Tests, which are very valuable anyway.

Reply:
For testing, means that this constructor is not used in xDrip code, but rather in testing (allowing to create fake points).

Reply:
In that case a mock class should be used.